framework/src/util/Utils.cpp

```diff
@@ -98,6 +98,31 @@ bool IsBundleFile(const std::string& location)
   }
 }
 
+bool OnlyContainsManifest(const std::string& location)
+{
+  if (location.empty()) {
+    throw std::runtime_error("Invalid (empty) location provided.");
+  }
+
+  BundleResourceContainer resContainer(location);
+  auto topLevelDirs = resContainer.GetTopLevelDirs();
+  return std::all_of(
```

Comment:
Could this be made more clear by writing it out rather than nested uses of using std::all_of??